container_files/demos/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,1583 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Virtual manipulation of datasets tutorial\n",
+    "\n",
+    "In this tutorial, we will be using 2 [Dataset types](../../../../doc/#builtin/datasets/Datasets.md.html), namely [Sampled Dataset](../../../../doc/#builtin/datasets/SampledDataset.md.html) and [Transposed Dataset](../../../../doc/#builtin/datasets/TransposedDataset.md.html). We will be using a manually created dataset that has two columns. The first column contains the name of machine learning concepts. The second column contains the description (i.e. first paragraph written about the concept in [Wikipedia](https://en.wikipedia.org))."
```

Comment:
peut-être ajouter un paragraphe qui dit qqchose du genre: this tutorial assumes you are familiar the the concept of procedures in MLDB. We recommend you go through the procedures and functions (link) tutorial before reading this tutorial ?